spd_db_mysql.cc

```diff
@@ -12872,6 +12886,7 @@ int spider_mysql_handler::append_order_by(
           DBUG_RETURN(HA_ERR_OUT_OF_MEM);
         str->q_append(SPIDER_SQL_COMMA_STR, SPIDER_SQL_COMMA_LEN);
       }
+*/
```

Comment:
my mistake, it's not needed.
I am sorry, as I do not have your latest server layer code.  I remove it for compile.